rule-packs/critical-assets.json

```
@@ -34,5 +34,17 @@
       }
     ],
     "alertLevel": "CRITICAL"
+  },
+  {
+    "name": "critical-assets-problematic",
+    "description": "Find critical assets that are most problematic.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND #CriticalAsset AS e THAT HAS AS r Problem          \n RETURN \n e._type, e.displayName, COUNT(r) AS numProblems            \n ORDER BY numProblems DESC",
```

```suggestion
        "query": "FIND #CriticalAsset AS e THAT HAS AS r Problem RETURN e._type, e.displayName, COUNT(r) AS numProblems ORDER BY numProblems DESC",
```
easier to not have new lines, unless we are confident in how they will render in a rule pack